posthog/celery.py

```diff
@@ -333,6 +334,13 @@ def setup_periodic_tasks(sender: Celery, **kwargs):
         name="sync datawarehouse sources that have settled in s3 bucket",
     )
 
+    # Every 2 hours try to retrieve and calculate total rows synced in period
+    sender.add_periodic_task(
+        crontab(hour="*/2"),
```

Comment:
buug alert, the default for everything is '*', so this is running every minute, every 2nd hour.

Reply:
```suggestion
        crontab(minute="0", hour="*/2"),
```